.github/ISSUE_TEMPLATE/content-bug.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Content bug
```

Comment:
Should we make it 

```suggestion
name: Translated content bug
```
?

Reply:
Oh I thought it was a word meaning content, but it was a repository name! But you merged, I guess I was right!